biz.aQute.http.testservers/src/aQute/http/testservers/Httpbin.java

```diff
@@ -166,7 +166,9 @@ public void _index(Request rq, Response rsp) throws IOException {
 	public void _redirect(Request rq, Response rsp, int count, int response) throws URISyntaxException {
 		if (count > 0) {
 			System.out.println("redirect " + count);
-			rsp.headers.put("Location", getBaseURI() + "/redirect/" + (--count) + "/" + response);
+			String location = (!rq.args.containsKey("relative") ? getBaseURI() : "") + "/redirect/" + (--count) + "/"
```

Comment:
Shouldn't this be more than just `containsKey` and instead check the value for truthiness? In the test case you use `?relative=true`. So there seems a small disconnect there.

Reply:
There are no other possible values for key "relative". But AFAIK, the syntax for the query part must be "key=value", even if the value doesn't matter.

Reply:
An alternate would be a different path `/redirectrelative/`

Reply:
Maybe, I'm not as deep in your test server design as you. But I don't like duplicating code and this seemed the easiest way to modify the behavior of _redirect in that particular aspect.